### PR TITLE
Add code samples for v1.4.0

### PR DESCRIPTION
**Pull request by curquiza**
Related to https://github.com/meilisearch/documentation/issues/2556

### CODE REVIEW

File: .code-samples.meilisearch.yaml [hunk 1]
```diff
@@ -699,3 +699,21 @@ multi_search_1: |-
           ->setIndexUid('movie_ratings')
           ->setQuery('us')
     ]);
+get_dictionary_1: |-
+  $client->index('books')->getDictionary();
```
Comment (alallema) [inline .code-samples.meilisearch.yaml:703]:
```suggestion
  $client->index('books')->getWordDictionary();
```

File: .code-samples.meilisearch.yaml [hunk 2]
```diff
@@ -699,3 +699,21 @@ multi_search_1: |-
...
+update_dictionary_1: |-
+  $client->index('books')->updateDictionary(['J. R. R.', 'W. E. B.']);
+reset_dictionary_1: |-
+  $client->index('books')->resetDictionary();
```
Comment (alallema) [inline .code-samples.meilisearch.yaml:707]:
```suggestion
  $client->index('books')->resetWordDictionary();
```

**Review by alallema — COMMENTED** [review]
Great! Just a bad naming hanging around

**Review by alallema — APPROVED** [review]
LGTM! 🌮

**Comment by brunoocasali** [comment]
bors merge

**Comment by curquiza** [comment]
bors merge

**Comment by curquiza** [comment]
Oh course it cannot work with the updated test...
merging manually

**Pull request merged**